interface-definitions/vpn_ipsec.xml.in

```diff
@@ -622,6 +622,33 @@
                   <valueless/>
                 </properties>
               </leafNode>
+              <leafNode name="flexvpn">
+                <properties>
+                  <help>Allow FlexVPN vendor ID payload (IKEv2 only)</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+              <leafNode name="virtual-interface">
```

Comment:
Any reason to not use `#include <include/generic-interface.xml.i>` instead?

Reply:
> Any reason to not use `#include <include/generic-interface.xml.i>` instead?

Done